.github/workflows/build.yml

```diff
@@ -7,6 +7,11 @@ jobs:
   build:
     name: Build
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch:
+          - amd64
+          - arm64
```

Comment:
Having this `strategy` will trigger the build job 2 times, each time with different value. We want to trigger image build once with 2 values provided to the `LINUX_ARCH`.

```suggestion
```